web_widget_number_ux_choice/static/src/js/number_ux_choice.js

```diff
@@ -0,0 +1,74 @@
+/**
+* Copyright (C) 2019 - Today: GRAP (http://www.grap.coop)
+* @author: Quentin DUPONT <quentin.dupont@grap.coop>
+* License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html) **/
+
+odoo.define('web.web_widget_number_ux_choice', function(require) {
+    "use strict";
+
+    var basic_fields = require('web.basic_fields');
+    var Registry = require('web.field_registry');
+    // var session = require('web.session');
+    // var rpc = require('web.rpc');
+    // var pyUtils = require("web.py_utils");
+
+    /*
```

Comment:
```suggestion
    /**
```

Marks the doc as a JSDoc string. Also the linter validates it using https://github.com/OCA/pylint-odoo/blob/master/pylint_odoo/examples/.jslintrc

Reply:
Done, thanks